packages/lib/src/components/Giftcard/Giftcard.tsx

```diff
@@ -74,6 +74,15 @@ export class GiftcardElement extends UIElement {
         }
     };
 
+    protected handleOrder = (order: Order) => {
+        this.elementRef._parentInstance.update({ order });
```

Comment:
Could just call `super.handleOrder(order)` here

Reply:
Because there is no super.handleOrder. I did think of this, but since we are assigning an anonymous function to the object state, we can't use super, because props that get overwritten.

Reply:
More specifically doing it that way results in the following:
```
Cannot read properties of undefined (reading 'call') 
```